packages/web-components/src/components/cbp-structured-list/cbp-structured-list.scss

```diff
@@ -80,7 +80,7 @@ cbp-structured-list {
     font-weight: var(--cbp-font-weight-bold);
 
     // Nesting 2 flex contexts will not take up 100% of the width
-    *:only-child {
+     *:only-child:not(cbp-checkbox):not(input[type="checkbox"]) {
```

Comment:
The checkbox should never be the only child. I'm not really sure why this is needed.

Reply:
adding the label to the check box seems to have made this change unnecessary, removing the not()'s and reverting to original selector. 

originally the global selector was applying it to the checkbox, & input of the checkbox so the checkbox was stretching to ~40% of the available width